vips/vip-501/bsctestnet.ts

```diff
@@ -0,0 +1,329 @@
+      },
+      {
+        target: unichainsepolia.RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3],bool))",
+        params: [
+          [
+            weETH.address,
+            [WETH_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+            [true, false, false],
+            false,
+          ],
+        ],
+        dstChainId: LzChainId.unichainsepolia,
+      },
+
+      // Market configurations
+      {
+        target: weETHMarket.vToken.address,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["86400"],
+        dstChainId: LzChainId.unichainsepolia,
+      },
+      {
+        target: weETH.address,
+        signature: "faucet(uint256)",
+        params: [weETHMarket.initialSupply.amount],
+        dstChainId: LzChainId.unichainsepolia,
+      },
+      {
+        target: weETH.address,
+        signature: "approve(address,uint256)",
+        params: [unichainsepolia.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.unichainsepolia,
+      },
+      {
+        target: weETH.address,
+        signature: "approve(address,uint256)",
+        params: [unichainsepolia.POOL_REGISTRY, weETHMarket.initialSupply.amount],
+        dstChainId: LzChainId.unichainsepolia,
+      },
+
+      {
+        target: unichainsepolia.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            weETHMarket.vToken.address,
+            weETHMarket.riskParameters.collateralFactor, // CF
+            weETHMarket.riskParameters.liquidationThreshold, // LT
+            weETHMarket.initialSupply.amount, // initial supply
+            unichainsepolia.NORMAL_TIMELOCK, // vToken receiver
+            weETHMarket.riskParameters.supplyCap, // supply cap
+            weETHMarket.riskParameters.borrowCap, // borrow cap
+          ],
+        ],
+        dstChainId: LzChainId.unichainsepolia,
+      },
+      {
+        target: weETHMarket.vToken.address,
+        signature: "transfer(address,uint256)",
+        params: [ethers.constants.AddressZero, weETHMarket.initialSupply.vTokensToBurn],
+        dstChainId: LzChainId.unichainsepolia,
+      },
+      (() => {
+        const vTokensMinted = convertAmountToVTokens(weETHMarket.initialSupply.amount, weETHMarket.vToken.exchangeRate);
+        const vTokensRemaining = vTokensMinted.sub(weETHMarket.initialSupply.vTokensToBurn);
+        return {
+          target: weETHMarket.vToken.address,
+          signature: "transfer(address,uint256)",
+          params: [weETHMarket.initialSupply.vTokenReceiver, vTokensRemaining],
+          dstChainId: LzChainId.unichainsepolia,
+        };
+      })(),
+      {
+        target: weETHMarket.vToken.address,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [weETHMarket.riskParameters.protocolSeizeShare],
+        dstChainId: LzChainId.unichainsepolia,
+      },
```

Comment:
Unnecessary, because the current value is already correct

Reply:
https://github.com/VenusProtocol/vips/pull/561/commits/a307d272d9a8bb706bf64850057229e97c421d4b